packages/kit-bg/src/vaults/impls/sol/Vault.ts

```
@@ -1013,19 +1014,50 @@ export default class Vault extends VaultBase {
+    encodedTx,
+    swapInfo,
+  }: {
+    encodedTx: IEncodedTxSol;
+    swapInfo?: ISwapTxInfo;
+  }) {
     const accountAddress = await this.getAccountAddress();
 
+    let newEncodedTx = encodedTx;
+
+    // internal okx sol swap tx need to replace recentBlockhash
+    if (swapInfo && swapInfo.swapBuildResData.OKXTxObject) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)